src/web_app/js/call.js

```diff
@@ -228,6 +231,9 @@ Call.prototype.onRemoteHangup = function() {
   // On remote hangup this client becomes the new initiator.
   this.params_.isInitiator = true;
 
+  // Let the callstats backend know we are closing the call.
+  this.pcClient_.sendCallstatsEvents('fabricTerminated');
```

Comment:
PC client should probably automatically send the terminated event when .close is called.